docs/run-move-locally.md

```diff
@@ -0,0 +1,278 @@
+---
+id: run-move-locally
+title: Run Move Programs Locally
+---
+<blockquote class="block_note">
+
+**Note:** Currently, you can run custom Move modules and scripts on a local network only, and not on the Libra testnet.
+</blockquote>
+
+This tutorial guides you through publishing a Move module and executing a Move transaction script on a local blockchain. To perform operations that are not natively supported by the existing Move transaction scripts, you can create and publish Move modules and write scripts to use these modules. For basic information on Move, refer to [Getting Started with Move](move-overview.md). For deeper technical understanding of Move, refer to the [technical paper](move-paper.md). For guidance on running a local network of nodes, refer to [Run a Local Network](run-local-network.md). The Libra CLI client provides the `dev` command to compile, publish, and execute Move programs locally. Refer to the [CLI Guide - dev command](reference/libra-cli#dev-d-mdash-operations-related-to-move-transaction-scripts-and-modules) for command usage. To see the list of subcommands,  enter `dev` on the CLI.
+
+To create, compile, and publish Move modules to an account on the local blockchain, follow the instructions in [compile and publish Move modules](#compile-and-publish-custom-move-module). To compile and execute a Move transaction script, follow the instructions in [compile and execute transaction scripts](#compile-and-execute-custom-transaction-script).
```

Comment:
Looks like the anchor links here are broken--this should fix it

```suggestion
To create, compile, and publish Move modules to an account on the local blockchain, follow the instructions in [compile and publish Move modules](#compile-and-publish-custom-move-modules). To compile and execute a Move transaction script, follow the instructions in [compile and execute transaction scripts](#compile-and-execute-custom-transaction-scripts).
```

Reply:
Fixed the anchor links and updated the PR.